.travis.yml

```diff
@@ -27,6 +27,24 @@ matrix:
         - brew update
         - brew outdated python || brew upgrade python
 
+    - os: windows
+      language: shell
+      env: VS100COMNTOOLS=$VS140COMNTOOLS
```

Comment:
VS100COMNTOOLS=$VS140COMNTOOLS forces to use Visual studio 14 instead of Visual Studio 10. 

But if we do not try to support Python 3.4 it is not required.